internal/support/scheduler/schedulecontext.go

```diff
@@ -14,6 +14,10 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/v2/models"
 )
 
+const (
+	TIMELAYOUT = "20060102T150405"
```

Comment:
can we call this TIMELAYOUT  thing as `SCHEDULER_TIME_FORMAT` ?

Reply:
Renamed and used proper PascalCase.